eclass/ruby-fakegem.eclass

```diff
@@ -61,14 +66,7 @@ RUBY_FAKEGEM_TASK_TEST="${RUBY_FAKEGEM_TASK_TEST-test}"
 #  - rdoc (calls `rdoc-2`, adds dev-ruby/rdoc to the dependencies);
 #  - yard (calls `yard`, adds dev-ruby/yard to the dependencies);
 #  - none
-case ${EAPI} in
-	5|6)
-		RUBY_FAKEGEM_RECIPE_DOC="${RUBY_FAKEGEM_RECIPE_DOC-rake}"
-		;;
-	*)
-		RUBY_FAKEGEM_RECIPE_DOC="${RUBY_FAKEGEM_RECIPE_DOC-rdoc}"
-		;;
-esac
+: ${RUBY_FAKEGEM_RECIPE_DOC:=rdoc}
```

Comment:
Haven't looked at changes much yet, but I did a metadata diff for non-EAPI=6 ebuild to see if anything changed, and some ruby-fakegem ebuilds came up suddenly gaining a USE="doc" with this change (like dev-ruby/railties-7.1.3.4). Thing is that they set a empty RUBY_FAKEGEM_RECIPE_DOC="" to cancel it.

```suggestion
: "${RUBY_FAKEGEM_RECIPE_DOC=rdoc}"
```
(also fwiw, we've started quoting `: ""` to avoid globbing ddos if something sets the variable to `*/*/*/*...` -- not that it's a big concern but it doesn't hurt, shellcheck also warns about that)